<ui version="4.0" >
 <class>ProxiesPageBase</class>
 <widget class="QWidget" name="ProxiesPageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Proxy Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <widget class="QComboBox" name="type" >
     <item>
      <property name="text" >
       <string>No proxies</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Auto configure proxies</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Set proxies</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="autopanel" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="autoconfig_label" >
        <property name="text" >
         <string>Config:</string>
        </property>
        <property name="buddy" >
         <cstring>autoconfig</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="autoconfig" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="setpanel" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="ftp_port" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4" >
       <widget class="QLabel" name="noproxies_label" >
        <property name="text" >
         <string>No proxies for:</string>
        </property>
        <property name="buddy" >
         <cstring>noproxies</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLineEdit" name="ftp_host" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLineEdit" name="http_host" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4" >
       <widget class="Line" name="Line1" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4" >
       <widget class="QLabel" name="ftp_label" >
        <property name="text" >
         <string>FTP proxy:</string>
        </property>
        <property name="buddy" >
         <cstring>ftp_host</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QSpinBox" name="http_port" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4" >
       <widget class="QLabel" name="http_label" >
        <property name="text" >
         <string>HTTP proxy:</string>
        </property>
        <property name="buddy" >
         <cstring>http_host</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4" >
       <widget class="QLineEdit" name="noproxies" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>type</tabstop>
  <tabstop>autoconfig</tabstop>
  <tabstop>http_host</tabstop>
  <tabstop>http_port</tabstop>
  <tabstop>ftp_host</tabstop>
  <tabstop>ftp_port</tabstop>
  <tabstop>noproxies</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
